infrastructure/transform/feature-data-store.tf

```diff
@@ -216,3 +216,41 @@ resource "azurerm_private_endpoint" "sql_server_features_pe" {
     private_dns_zone_ids = [azurerm_private_dns_zone.sql.id]
   }
 }
+
+resource "azuread_group" "ad_group_apps" {
+  display_name     = "${var.naming_suffix} flowehr-apps"
+  owners           = [data.azurerm_client_config.current.object_id]
+  security_enabled = true
+}
+
+resource "azuread_group" "ad_group_developers" {
+  display_name     = "${var.naming_suffix} flowehr-developers"
+  owners           = [data.azurerm_client_config.current.object_id]
+  security_enabled = true
+}
+
+resource "azuread_group" "ad_group_data_scientists" {
+  display_name     = "${var.naming_suffix} flowehr-data-scientists"
+  owners           = [data.azurerm_client_config.current.object_id]
+  security_enabled = true
+}
+
+output "apps_ad_group_display_name" {
```

Comment:
Suggest moving these to `outputs.tf`

Reply:
Done 8cbba0c21ea5058129e27fc2f66482b6901cafce